ale_linters/ruby/rubocop.vim

```diff
@@ -0,0 +1,45 @@
+if exists('g:loaded_ale_linters_ruby_rubocop')
```

Comment:
I'll add Author comments to the top of this file and others to credit you. Let me know if there is a preferred name you would like in the files via email or just comments on GitHub.